openedx/core/djangoapps/content/search/tasks.py

```diff
@@ -116,6 +116,19 @@ def update_library_components_collections(library_key_str: str, collection_key:
     api.update_library_components_collections(library_key, collection_key)
 
 
+@shared_task(base=LoggedTask, autoretry_for=(MeilisearchError, ConnectionError))
+@set_code_owner_attribute
+def update_library_containers_collections(library_key_str: str, collection_key: str) -> None:
```

Comment:
https://github.com/openedx/edx-platform/pull/36476 started using a single `LibraryCollectionLocator` key wherever we could, instead of passing the `library_key` with the `collection_key`. Can you do the same thing here?

Reply:
Fixed!